Unity.gitignore

```diff
@@ -1,36 +1,37 @@
-[Ll]ibrary/
-[Tt]emp/
-[Oo]bj/
-[Bb]uild/
-[Bb]uilds/
-Assets/AssetStoreTools*
+**/[Ll]ibrary/
+**/[Tt]emp/
+**/[Oo]bj/
+**/[Bb]uild/
+**/[Bb]uilds/
+**/Assets/AssetStoreTools*
 
-# Visual Studio cache directory
-.vs/
+# Visual Studio / Visual Studio Code cache directory
```

Comment:
This isn't a cache directory, and we have an existing template to show what can be committed in  [`Global\VisualStudioCode.gitignore`](https://github.com/github/gitignore/blob/master/Global/VisualStudioCode.gitignore)